core/src/test/scala/com/rallyhealth/vapors/core/evaluator/LogicalOperatorSpec.scala

```diff
@@ -0,0 +1,168 @@
+package com.rallyhealth.vapors.core.evaluator
+
+import com.rallyhealth.vapors.core.Example.JoeSchmoe
+import com.rallyhealth.vapors.core.data.{Fact, FactsMatch, NoFactsMatch}
+import com.rallyhealth.vapors.core.dsl
+import org.scalatest.wordspec.AnyWordSpec
+
+class LogicalOperatorSpec extends AnyWordSpec {
```

Comment:
Split out from other tests, and converted to address DSL's vararg requirement of at least 2 arguments for `and` / `or`...

I should probably bypass the DSL and test the expressions on empty lists and pairs.